go.mod

```diff
@@ -26,6 +26,7 @@ require (
 	github.com/moby/sys/mountinfo v0.6.2
 	github.com/moby/sys/signal v0.7.0
 	github.com/morikuni/aec v1.0.0
+	github.com/muesli/termenv v0.15.1
```

Comment:
I could get rid of this lib and just handle the escape sequence in an util package.

Reply:
Done